integration/integrate-with-servicenow.adoc

```diff
@@ -0,0 +1,22 @@
+:_content-type: ASSEMBLY
+[id="integrate-with-servicenow"]
+= Integrating with ServiceNow
+include::modules/common-attributes.adoc[]
+:context: integrate-with-servicenow
+
+toc::[]
+
+[role="_abstract"]
+You can configure {product-title-short} to send events to ServiceNow by configuring a generic webhook integration in {product-title-short}.
+
+The following steps represent a high-level workflow for integrating {product-title-short} with ServiceNow:
+
+. In ServiceNow, configure a REST API endpoint to use in {product-title-short}. See link:https://www.redhat.com/en/blog/how-integrate-red-hat-advanced-cluster-security-kubernetes-servicenow[How to integrate Red Hat Advanced Cluster Security for Kubernetes with ServiceNow] for more information.
```

Comment:
I think we're supposed to confirm with support that this link is okay.

Reply:
Confirmed with Support that this link and the other link are OK.